### PR TITLE
docs: improve syntax, grammar, punctuation

### PR DESCRIPTION
**Pull request by kmdrGroch**

### CODE REVIEW

**Comment by kmdrGroch** [comment]
Sorry, it took so long, but as I told you I've started studying and I have much less time.

File: src/stryker4s/index.pug [hunk 1]
```diff
@@ -9,7 +9,7 @@ block content
 
         Currently Stryker4s is still in early development, which means it's propably not ready for large projects just yet.
 
-        But, this shouldn't hold you back to give Stryker4s a try. 
+        But this shouldn't hold you back to give Stryker4s a try.
```
Comment (hugo-vrijswijk) [inline src/stryker4s/index.pug]: I don't think the comma is needed here. Maybe we should change it to "However, this shouldn't hold you back to give Stryker4s a try."

File: src/index.pug [hunk 2]
```diff
@@ -83,9 +84,9 @@ block content
                 /* 4 */ return true;
                 ```
 
-                We call those modifications mutants. After the mutants have been found, they are applied one by one and your tests will be executed against them. 
-                If at least one of your tests fail, we say the mutant is *killed*. That's what we want! If no tests fail, it *survived*. 
-                The better your tests, the less mutants survive.
+                We call those modifications mutants. After the mutants have been found, they are applied one by one and your tests are executed against them. 
+                If at least one of your tests fail, we say the mutant is *killed*. That's what we want! If no test fail, it *survived*.
```
Comment (hugo-vrijswijk) [inline src/index.pug]: I don't think the change is correct now. It should be "If no test fails"

File: src/faq.pug [hunk 3]
```diff
@@ -6,16 +6,16 @@ block content
             :markdown-it
                 # Frequently Asked Questions
                 Here you will find the answer to the most common issues first-time users of Stryker encounter.  
-                If this page does not answer your question, please feel free to [![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).
+                If this page doesn't answer your question, please feel free to [![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).
                 If you think you've encountered a bug, please also contact us on Gitter first, before submitting a bug report.
 
                 ## Q: Stryker is slow! How do I speed it up?  
-                **A: Improve your tests' performance**
```
Comment (hugo-vrijswijk) [inline src/faq.pug]: I think the two spaces at the end here tell markdown that the next line should be on a newline. So it is needed (but I can't check atm)

File: src/blog/2018-10-6/mutation-switching.pug [hunk 4]
```diff
@@ -28,25 +27,25 @@ block content
...
 
         1. Change `>` to `>=`
         1. Change `>` to `<`
         1. Change `>` to `==`
         
-        If we apply the mutation one by one we would need to compile the code base three times. 
+        If we apply the mutation one by one, we would need to compile the code base three times. 
         If we assume the compile time of this program is 10 seconds, we already have 30 seconds of compile time for one full mutation run. 
-        This would quickly get out of hand when the code base is bigger and generates more mutants.
+        This would quickly get out of hand that bigger code base generates more mutants.
```
Comment (hugo-vrijswijk) [inline src/blog/2018-10-6/mutation-switching.pug]: Don't think this is a correct sentence

File: src/index.pug (hunk 2 above)
Comment (kmdrGroch) [reply src/index.pug]: indeed. Haven't noticed

File: src/stryker4s/index.pug (hunk 1 above)
Comment (kmdrGroch) [reply src/stryker4s/index.pug]: Sounds great

File: src/faq.pug (hunk 3 above)
Comment (kmdrGroch) [reply src/faq.pug]: I have made a dot in the first commit, I haven't noticed that I removed space in the second commit

File: src/blog/2018-10-6/mutation-switching.pug (hunk 4 above)
Comment (kmdrGroch) [reply src/blog/2018-10-6/mutation-switching.pug]: I must have misunderstood that sentence

**Comment by hugo-vrijswijk** [comment]
Oops. I kind of forgot about your pull request. Sorry! As far as I can see this is good to merge! 👍

**Comment by nicojs** [comment]
@kmdrGroch thanks for this awesome PR! Sentences have better construction in general and a lot of typo's were still in there. Thanks to you and your friend! ❤️

**Comment by kmdrGroch** [comment]
np @nicojs :) as long as my PRs are merged I am happy xD

**Pull request merged**